doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,9 +1,27 @@
 FORMAT: 1A
-HOST: http://orion.lab.fiware.org/
+HOST: http://telefonicaid.github.io/fiware-orion/api/v2/
```

Comment:
I though that HOST was for the "reference host" holding the API implementation, not the URL hosting the apiary rendering of the spec. Could you clarify, please? Maybe a link to the piece of doc about HOST parameter in apiary documentaiton could help.